Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -11,7 +11,7 @@
 //  This exposes internal functionality which may cause unexpected behavior if used directly.
 import Contacts
 import PassKit
-import StripePaymentSheet
+@_spi(STP) import StripePaymentSheet
```

Comment:
I'm guessing this is temporary? We gaveled on a way to do betas in the mobile SDK -- i will link you via slack

Reply:
This won't be a beta, I'm just using `@_spi` to gate it during development.